packages/vite-plugin-cloudflare/src/utils.ts

```diff
@@ -1,5 +1,27 @@
+import * as path from 'node:path';
 import { invariant } from './shared';
-import type { WorkerOptions } from 'miniflare';
+import type { SharedOptions, WorkerOptions } from 'miniflare';
+
+const DEFAULT_PERSIST_PATH = '.wrangler/state/v3';
+
+export function getPersistence(
+	persistTo: string | false | undefined,
+	root: string,
+): Partial<SharedOptions> {
```

Comment:
`Partial<SharedOptions>` seems a bit broad to me, what about narrowing it down?

```suggestion
type PersistOptions = Pick<
	SharedOptions,
	| 'cachePersist'
	| 'd1Persist'
	| 'durableObjectsPersist'
	| 'kvPersist'
	| 'r2Persist'
>;

export function getPersistence(
	persistTo: string | false | undefined,
	root: string,
): Partial<PersistOptions> {
```